packages/@aws-cdk/aws-lambda-event-sources/test/test.kafka.ts

```diff
@@ -84,7 +137,7 @@ export = {
     },
   },
 
-  'self managed kafka': {
+  'self-managed kafka': {
```

Comment:
can you update the 'default' case to not use & assert `secret`? 

The 'default' test case is intended to assert that the expected defaults are configured. This means that they must only use all the required props.

Reply:
This means `'default'` will throw an error since either `secret` or `vpc` is required. I've made the change.

Reply:
The default test case must specify all the required properties and then verify that the other properties correctly configure the expected defaults. So you will need to change this to match up with that.

Reply:
I'm not sure I understand what you're asking for. Do you mean "required" by TypeScript? If so, that's what this test is doing. `secret` and `vpc` are both optional parameters because only one or the other is required. If either is missing, I'm not aware of any default we could safely choose.

Reply:
Let me try again and see if I can make it clearer.

The 'default' test case is testing that defaults for optional parameters are set as expected. In your case 'secret' or 'vpc' is required.

So I would suggest configuring one of them and then asserting that the rest of the optional parameters (excluding 'secret') is configured as expected.

The test case as you have right now is required, but it shouldn't be called the 'default' test case and is separate from it.

Is that clearer?

Reply:
Thanks, that clears it up. I updated the `default` test.